policykit/integrations/discourse/tasks.py

```diff
@@ -114,17 +113,3 @@ def discourse_listener_actions():
                                 vote.save()
                         else:
                             b = BooleanVote.objects.create(proposal=proposed_action.proposal, user=u, boolean_value=val)
-
-            # Update proposal
-            logger.info('updating proposal')
-            for policy in PlatformPolicy.objects.filter(community=community):
-                logger.info('about to filter policy')
-                if filter_policy(policy, proposed_action):
-                    logger.info('just filtered policy')
-                    cond_result = check_policy(policy, proposed_action)
-                    if cond_result == Proposal.PASSED:
-                        logger.info('passed proposal discourse')
-                        pass_policy(policy, proposed_action)
-                    elif cond_result == Proposal.FAILED:
-                        logger.info('failed proposal discourse')
-                        fail_policy(policy, proposed_action)
```

Comment:
@rickygv99 I think this can be removed entirely because the celery task at `policyengine/tasks.py` already re-evaluates all `proposed` PlatformActions. Am I missing something?

Reply:
Yes, I think we should be okay to remove it!